### PR TITLE
Included some libs and fixed mistoken chars

### PR DESCRIPTION
**Pull request by EduApps-CDG**
That's my conttibution

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -203,6 +203,13 @@
     "image": "https://user-images.githubusercontent.com/10347617/73778440-479a8280-479c-11ea-8446-1ab4b0f5c7fd.png",
     "language": "PHP"
   },
+  "LibGDX": {
+    "imports": ["com.badlogic.gdx"],
+    "technologies": ["Java Frameworks", "Mobile Applications", "Game Development", "Desktop Applications"],
+    "description": "libGDX is a free and open-source game-development application framework written in the Java programming language with some C and C++ components for performance dependent code. It allows for the development of desktop and mobile games by using the same code base. It is cross-platform, supporting Windows, Linux, Mac OS X, Android, iOS, BlackBerry and web browsers with WebGL support.",
+    "image": "https://www.badlogicgames.com/wordpress/wp-content/uploads/2011/05/libGDX-RedGlossyNoReflection.png",
```
Comment (EduApps-CDG) [inline libraries.json:210]: Added [LibGDX](https://badlogicgames.com)

File: libraries.json [hunk 2]
```diff
@@ -203,6 +203,13 @@
...
+    "language": "Java"
+  },
```
Comment (EduApps-CDG) [inline libraries.json:212]: Added [LibGDX](https://badlogicgames.com)

File: libraries.json [hunk 3]
```diff
@@ -217,12 +224,12 @@
...
-		"technologies": ["Web Development", "Microservices", "Java Frameworks],
-		"description": "Micronaut is a modern, JVM-based, full-stack framework for building modular, easily testable microservice and serverless applications.",
-		"image": "https://raw.githubusercontent.com/micronaut-projects/static-website/master/main/src/main/resources/assets/images/micronaut_mini.svg",
-	},
+    "language": "Java",
+	"imports": ["io.micronaut"],
+	"technologies": ["Web Development", "Microservices", "Java Frameworks"],
+	"description": "Micronaut is a modern, JVM-based, full-stack framework for building modular, easily testable microservice and serverless applications.",
+	"image": "https://raw.githubusercontent.com/micronaut-projects/static-website/master/main/src/main/resources/assets/images/micronaut_mini.svg",
+  },
```
Comment (EduApps-CDG) [inline libraries.json:232]: Added mistoken characters

File: libraries.json [hunk 4]
```diff
@@ -196,13 +196,27 @@
     "image": "https://kivy.org/logos/kivy-logo-black-256.png",
     "language": "Python"
   },
+  "KiwiJS": {
+    "imports": ["kiwi-js"],
+    "technologies": ["Web Development", "Mobile Development", "Game Development"],
+    "description": "Kiwi.js is a mobile first HTML5 and Javascript game engine supporting both Canvas and WebGL rendering.",
+    "image": "http://www.kiwijs.org/wp-content/themes/kiwijs/assets/logo.png",
+    "language": "JavaScript"
+  },
```
Comment (EduApps-CDG) [inline libraries.json:205]: Added [KiwiJS](https://kiwijs.org)

**Comment by peti2001** [comment]
thanks for the fixes and for the new libraries

**Comment by peti2001** [comment]
Hi @EduApps-CDG,

We are planning to rewrite the [repo_info_extractor](https://github.com/codersrank-org/repo_info_extractor/issues/160). If you have tried it before please have a look at our plans and feel free to share your thoughts https://github.com/codersrank-org/repo_info_extractor/issues/160.

**Comment by EduApps-CDG** [comment]
> Hi @EduApps-CDG,
> 
> We are planning to rewrite the [repo_info_extractor](https://github.com/codersrank-org/repo_info_extractor/issues/160). If you have tried it before please have a look at our plans and feel free to share your thoughts [codersrank-org/repo_info_extractor#160](https://github.com/codersrank-org/repo_info_extractor/issues/160).

Ill take a look!

**Pull request merged**